_inc/lib/debugger/class-jetpack-debug-data.php

```diff
@@ -173,33 +181,39 @@ public static function debug_data() {
 		$tokenset = '';
 		if ( $blog_token ) {
 			$tokenset = 'Blog ';
+			$blog_key = substr( $blog_token, 0, strpos( $blog_token, '.' ) );
+			// Intentionally not translated since this is helpful when sent to Happiness.
+			$blog_key = ( $blog_key ) ? $blog_key : 'Potentially Malformed Token.';
 		}
 		if ( $user_token ) {
 			$tokenset .= 'User';
+			$user_key  = substr( $user_token, 0, strpos( $user_token, '.' ) );
+			// Intentionally not translated since this is helpful when sent to Happiness.
+			$user_key = ( $user_key ) ? $user_key : 'Potentially Malformed Token.';
 		}
 		if ( ! $tokenset ) {
 			$tokenset = 'None';
 		}
 
 		$debug_info['current_user'] = array(
 			'label'   => 'Current User',
-			'value'   => $user_id,
+			'value'   => self::human_readable_user( $user_id ),
 			'private' => false,
 		);
 		$debug_info['tokens_set']   = array(
-			'label' => 'Tokens defined',
-			'value' => $tokenset,
-			'private => false,',
+			'label'   => 'Tokens defined',
+			'value'   => $tokenset,
+			'private' => false,
 		);
 		$debug_info['blog_token']   = array(
 			'label'   => 'Blog token',
```

Comment:
Can we call this (and User below) "Blog Token Key" or "Blog Token ID"? I want to avoid people saying "Jetpack just cut my blog token's length in half, which obviously means it's easier to brute force" or something :)

Reply:
Or just always add an `…` after the key?

Reply:
The `private => false` is part of the keys Core uses since this just feeds into the existing code via filter. I can check if they assume if none present or if they need them declared. I declared them to be clear the visibility.

I'll rename to "Blog Token Key" etc.

Reply:
Ah! I didn't realize it was part of core - let's leave it and be explicit, then.